README.md

```diff
@@ -128,6 +128,53 @@ You can directly extend and run the tests by modifying
 
     tests/test.hs
 
+### Parallelism in Tests
```

Comment:
The documentation looks very good. I propose we keep the details closer to the code by moving this section to a note in `test.hs`. But here we could say

```
### Parallelism in Tests

Most tests run in parallel, with a few module dependencies built sequentially in advance.
Benchmarks run sequentially after all other tests have finished. For detail on adding tests,
see note [bla] in `tests/test.hs`.
```